docker/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM continuumio/miniconda3:4.12.0
+
+LABEL maintainer="xcube-team@brockmann-consult.de"
+LABEL name=xcube_geodb_openeo
+
+ARG GEODB_OPENEO_VERSION=0.0.1.dev0
```

Comment:
Not used, but should be used. I'd prefer you create docker images from (dev) releases only.